code/modules/reagents/chemistry/recipes/food.dm

```diff
@@ -102,6 +102,22 @@
 	for(var/i = 1, i <= created_volume, i++)
 		new /obj/item/reagent_containers/food/snacks/dough(location)
 
+///Cookies by Ume
+
+/datum/chemical_reaction/cookiedough
+	name = "Dough"
+	id = "dough"
+	result = null
+	required_reagents = list("milk" = 10, "flour" = 10, "sugar" = 5)
+	result_amount = 1
+	mix_message = "The ingredients form a dough. It smells sweet and yummy."
+
+/datum/chemical_reaction/cookiedough/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
```

Comment:
Maybe should use `drop_location()` here. Otherwise people could become cookie dough factories by eating the constituent ingredients and have a bunch of dough pop out of them 😄

Reply:
_That's actually kinda intended so I wouldn't fix it, I just find it hilarious_

Reply:
🤔

Reply:
People can do this with normal dough, I'd leave it in for comedic value myself. :man_shrugging:

Reply:
I actually do it all the time in front of people when I'm the chef